spec/killbill/helpers/payment_plugin_spec.rb

```diff
@@ -308,6 +308,17 @@
       source.first_name.should == 'John'
       source.last_name.should == 'Doe'
     end
+
+    it "should support search_payments" do
+      response = ::Killbill::Test::TestResponse.create(
+        :api_call => :purchase, :kb_payment_id => @kb_payment_id,
+        :kb_account_id => @kb_account_id, :kb_tenant_id => @call_context.tenant_id,
+        :success => true, :created_at => Time.now, :updated_at => Time.now
+      )
+      results = plugin.search_payments(@kb_payment_id, 0, 100, [], @call_context).to_a
```

Comment:
The test doesn't look right. The [search_payments](https://github.com/killbill/killbill-plugin-api/blob/master/payment/src/main/java/org/killbill/billing/payment/plugin/api/PaymentPluginApi.java#L159) API returns a [Pagination](https://github.com/killbill/killbill-api/blob/master/src/main/java/org/killbill/billing/util/entity/Pagination.java) object, which doesn't have a `to_a` method (I'm guessing JRuby injects a `to_a` for convenience, provided `hasNext` and `next` are implemented).

The test should pass if you replace `to_a` with `.iterator.to_a`.

That being said, going back to the original [bug](https://github.com/killbill/killbill-java-parser/pull/9), did you manage to trigger the issue by calling the API from Kill Bill, or via a similar test (I see `./spec/ach_chase/base_plugin_spec.rb` in the backtrace)?